src/utilities/new_versions.jl

```diff
@@ -280,24 +295,69 @@ function make_pr_for_new_version(
             if commit_was_success
                 @info("Commit was a success")
                 api_retry() do
-                    @mock git_push("origin", new_branch_name; force=true, env=env)
+                    @mock git_push(
+                        "origin", new_branch_name, pkey_filename; force=true, env=env
+                    )
                 end
 
-                create_new_pull_request(
+                new_pr, _ = create_new_pull_request(
                     forge,
                     repo,
                     new_branch_name,
                     master_branch_name,
                     new_pr_title,
                     new_pr_body,
                 )
+
+                cc_user && cc_trigger_user(forge, repo, new_pr; env=env)
+                unsub_from_prs && unsub_from_pr(forge, new_pr)
             end
         end
     end
 
     return nothing
 end
 
+function cc_trigger_user(
```

Comment:
probably clearer to say 
```suggestion
function cc_mention_user(
```
?
I think both github and gitlab use this term?